docs/guides/server/keycloak-truststore.adoc

```diff
@@ -38,6 +38,8 @@ For HTTPS requests, this option verifies the hostname of the server's certificat
 * `ANY` means that the hostname is not verified.
 * `WILDCARD` allows wildcards in subdomain names, such as *.foo.com.
 * When using `STRICT`, the Common Name (CN) must match the hostname exactly.
++
+Please note that this settings does not apply to LDAP secure connections which requires strict hostname checking.
```

Comment:
```suggestion
Please note that this setting does not apply to LDAP secure connections, which require strict hostname checking.
```

Reply:
BTW, a subordinate clause that begins with "which" begins with a comma.

Reply:
Thank you!